25-minimize.ks

```diff
@@ -64,14 +62,20 @@ echo fdi > /etc/hostname
 echo " * locking root account"
 passwd -l root
 
+echo " * store list of packages sorted by size"
+rpm -qa --queryformat '%{SIZE} %{NAME}\n' | sort -n -r > /usr/PACKAGES-LIST
+
 echo " * cleaning up yum cache and removing rpm database"
-rpm -qa > /usr/packages_list
 yum clean all
 rm -rf /var/lib/{yum,rpm}/*
 # fix the vim syntax markup */
+
+# no more python loading after this step
+echo " * removing python precompiled *.pyc files"
+find /usr/lib64/python*/ /usr/lib/python*/ -name *py[co] -print0 | xargs -0 rm -f
 %end
 
 %post --nochroot
-echo " * disquieting the microkernel boot process"
+echo " * disquieting the boot process"
```

Comment:
Every bit counts. Actually, the backgrounds from line 42 were big deal in this patch.